sunpy/map/mapbase.py

```diff
@@ -181,6 +181,9 @@ class GenericMap(NDData):
     # in an error.  We explicitly set __array_ufunc__ = None so that the NumPy
     # call, and consequently the Quantity operator, will return NotImplemented.
     __array_ufunc__ = None
+    # This attribute is used for resolving which map source should be used if multiple
+    # source classes are identified for a given file. A higher number beats a lower number.
+    _priority = 100
```

Comment:
Should we just start with 10 or 1?

Reply:
🤷

Reply:
I'm open to whatever. I don't like 1 because you would potentially have to have negative priorities if you wanted more than one thing below this in priority which feels weird.

Reply:
Yeah I think 1 is too low.

Reply:
I would start with 0, negative makes perfect sense to me for "less than default".